app/baseapp.go

```diff
@@ -599,10 +599,6 @@ func (app *BaseApp) getContextForTx(mode RunTxMode, txBytes []byte) (ctx sdk.Con
 
 // Iterates through msgs and executes them
 func (app *BaseApp) runMsgs(ctx sdk.Context, msgs []sdk.Msg, mode RunTxMode) (result sdk.Result) {
-	if app.runMsg != nil {
-		return app.runMsg(ctx, msgs, mode)
-	}
-
```

Comment:
Should also remove:
`func (app *BaseApp) SetRunMsg(runMsg RunMsg)` and `runMsg           RunMsg`.